applications/api_test.py

```diff
@@ -66,6 +66,13 @@ def test_derive_application_state():
     app.refresh_from_db()
     assert derive_application_state(app) == AppStates.AWAITING_USER_SUBMISSIONS.value
 
+    # The resume requirement should be considered fulfilled if the user uploads a resume *or* provides a LinkedIn URL
+    app.resume_file = None
+    app.linkedin_url = "http://example.com/linkedin"
```

Comment:
The `linkedin_url` will be a non-null value by default for `BootcampApplicationFactory` (`linkedin_url = Faker("url")`) - so that is currently causing the test condition on line 62 to fail.  Either changing the factory linked_in url to be None by default or modifying these tests is fine with me.

Reply:
Hmph... don't know how I missed this. Fixed now!